cli/src/lib.rs

```diff
@@ -924,6 +930,15 @@ fn process_command(opts: Opts) -> Result<()> {
             address,
             idl,
         } => account(&opts.cfg_override, account_type, address, idl),
+        Command::Completions { shell } => {
+            generate(
```

Comment:
Nit: `generate` is pretty generic for this file — could you use its full path `clap_complete::generate` instead?
```suggestion
            clap_complete::generate(
```